### PR TITLE
for #1238 Make connectionId binding with backendConnection

### PR DESCRIPTION
**Pull request by cherrylzhao**
- simplify schema
- simplify commandPacketFactory

### CODE REVIEW

File: sharding-proxy/src/main/java/io/shardingsphere/shardingproxy/backend/BackendHandlerFactory.java [hunk 1]
```diff
@@ -53,68 +52,59 @@
     private static final GlobalRegistry GLOBAL_REGISTRY = GlobalRegistry.getInstance();
     
     /**
-     * Create new instance of text protocol backend handler.
+     * Create new com query backend handler instance by SQl judge.
```
Comment (terrymanu) [inline sharding-proxy/src/main/java/io/shardingsphere/shardingproxy/backend/BackendHandlerFactory.java]: SQl should be SQL

File: sharding-proxy/src/main/java/io/shardingsphere/shardingproxy/backend/BackendHandlerFactory.java [hunk 2]
```diff
@@ -53,68 +53,63 @@
     private static final GlobalRegistry GLOBAL_REGISTRY = GlobalRegistry.getInstance();
     
     /**
-     * Create new instance of text protocol backend handler.
+     * Create new instance by sql judge.
```
[the remark this answers is not in the document]
Comment (cherrylzhao) [reply sharding-proxy/src/main/java/io/shardingsphere/shardingproxy/backend/BackendHandlerFactory.java]: ok, I have fix it.

File: sharding-proxy/src/main/java/io/shardingsphere/shardingproxy/backend/jdbc/connection/BackendConnection.java [hunk 3]
```diff
@@ -94,14 +93,14 @@ public void setTransactionType(final TransactionType transactionType) {
     /**
      * Change logic schema of current channel.
      *
-     * @param currentSchema current schema
+     * @param schemaName current schema
```
[the remark this answers is not in the document]
Comment (cherrylzhao) [reply sharding-proxy/src/main/java/io/shardingsphere/shardingproxy/backend/jdbc/connection/BackendConnection.java]: ok, I have fixed it.

File: sharding-proxy/src/main/java/io/shardingsphere/shardingproxy/backend/BackendHandlerFactory.java (hunk 1 above)
Comment (cherrylzhao) [reply sharding-proxy/src/main/java/io/shardingsphere/shardingproxy/backend/BackendHandlerFactory.java]: ok, I have fixed it.

**Pull request merged**